src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/OmuContactController.kt

```diff
@@ -22,6 +23,10 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.OmuContact
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request.UpdateOmuEmailRequest
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.OmuService
 
+@Tag(
+  name = Tags.OMU_CONTACT_DETAILS,
+  description = "CVL stores contact information for Offender Management Units (OMUs). These endpoints are responsible for retrieving and managing that information",
```

Comment:
Can centralise the descriptions in the swagger bean config but as this is only used in one place I've just inlined

You could imagine having to keep the description across multiple annotations a bit annoying